presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Config.java

```diff
@@ -462,4 +471,17 @@ public HiveS3Config setRequesterPaysEnabled(boolean requesterPaysEnabled)
         this.requesterPaysEnabled = requesterPaysEnabled;
         return this;
     }
+
+    @NotNull
```

Comment:
S3 ignores anything after the first character, so we also should enforce it is exactly one character with size validation (in addition to not null)
```java
@Size(min = 1, max = 1)
```